pkg/controller/validatingadmissionpolicystatus/controller.go

```diff
@@ -131,7 +131,7 @@ func (c *Controller) processNextWorkItem(ctx context.Context) bool {
 		return true
 	}
 
-	utilruntime.HandleError(err)
+	utilruntime.HandleErrorWithContext(ctx, err, "Error processing validating admission policy work item", "item", key)
```

Comment:
I feel like we are using couple of different descriptions in similar places where the syncs fail. Maybe we should unify them?